src/map/script.c

```diff
@@ -9019,6 +9019,32 @@ BUILDIN(getequipisenableref)
 	return true;
 }
 
+/**
+ * Checks if the equipped item allows options.
+ * *getequipisenableopt(<equipment_index>);
+ * 
+ * @param equipment_index as the inventory index of the equipment.
+ * @return 1 on enabled 0 on disabled.
+ */
+BUILDIN(getequipisenableopt)
+{
+	int i = -1, index = script_getnum(st,2);
+	struct map_session_data *sd = script->rid2sd(st);
+	
+	if (sd == NULL)
```

Comment:
Probably here need add script_pushint(st, -1) or like this for prevent script from termination